integration-tests/utils/integration-utils.js

```diff
@@ -48,7 +48,7 @@ const getAnimatedDrawer = textContains => {
 // Note: isEnabled is not fully supported for native anchor and button elements (only aria-disabled is examined).
 const clickable = ({ text, textContains, isDescendant = false, isEnabled = true }) => {
   const checkEnabled = isEnabled === false ? '[@aria-disabled="true"]' : '[not(@aria-disabled="true")]'
-  const base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"])${checkEnabled}`
+  const base = `(//a | //button | //*[@role="button"] | //*[@role="link"] | //*[@role="combobox"] | //*[@role="option"] | //*[@role="tab"])${checkEnabled}`
```

Comment:
@petesantos and I were talking about this earlier. Adding another `or` to that logic makes it difficult to know which regular expression piece was targeted when a test fails due to timing out while waiting for that XPath. Can we pull out `//*[@role="tab"]` into its own function (i.e. `const tab = …`)?

Reply:
@kyuksel Sorry, but I'm not following what you are requesting. Can you provide more of an example?

Reply:
Below is a screenshot of a diff for an example:

![image](https://user-images.githubusercontent.com/5438223/157322872-cebc4e6a-ffd4-4808-a6da-f3a1ea14acb4.png)

Reply:
Thanks @kyuksel. 

I am struggling though to understand why we want a different method for "tab" because I thought that "clickable" is meant to encompass all elements that are clickable. When I wrote the test, I fully expected "clickable" to work, and I don't think I would have found a "tab" method.

Reply:
@petesantos I have a feeling you'll articulate it better 😊

Reply:
I read it as "I'm looking for a clickable thing with text 'Create new notebook'". I don't think it makes sense to split out `tab` by itself… if we want to go that route, we should also split out all the others into individual methods (combobox, select, etc.).

I **do** agree that we shouldn't add `img` here, because images are generally not clickable.